client/ayon_core/hosts/maya/api/lib_renderproducts.py

```diff
@@ -1136,6 +1136,21 @@ def get_render_products(self):
         light_groups_enabled = False
         has_beauty_aov = False
         colorspace = lib.get_color_management_output_transform()
+        global_aov_enabled = bool(
+            self._get_attr("redshiftOptions.aovGlobalEnableMode")
+        )
+        if not global_aov_enabled:
+            beauty_name = "BeautyAux" if has_beauty_aov else ""
+            for camera in cameras:
+                products.insert(0,
+                                RenderProduct(productName=beauty_name,
+                                            ext=ext,
+                                            multipart=self.multipart,
+                                            camera=camera,
+                                            colorspace=colorspace))
```

Comment:
```suggestion
            # only beauty output
            for camera in cameras:
                products.insert(0,
                                RenderProduct(productName="",
                                            ext=ext,
                                            multipart=self.multipart,
                                            camera=camera,
                                            colorspace=colorspace))
```
If there are no AOVs then `has_beauty_aov`  may never be True?

Also, I'd move `light_groups_enabled` and `has_beauty_aovs` to below this to make it clear that those vars are unrelated to this part of the code.